backend/src/Designer/Services/Implementation/AppDevelopmentService.cs

```diff
@@ -239,44 +239,42 @@ private async Task<string> GetTaskIdBasedOnLayoutSet(AltinnRepoEditingContext al
             return null;
         }
 
-        public async Task<LayoutSets> ConfigureLayoutSet(AltinnRepoEditingContext altinnRepoEditingContext,
-            string layoutSetName, CancellationToken cancellationToken = default)
+        public async Task<LayoutSets> UpdateLayoutSet(AltinnRepoEditingContext altinnRepoEditingContext, string layoutSetId,
```

Comment:
From what I understand we have 4 different cases in this method.
1. Not an layout-set app
2. Adding a new layout set configuration
3. Updating existing layout set configuration
4. Updating existing layout-set configuration with id change which will reflect underlying folder rename

Splitting those cases in methods will increase readability. I guess first case can be a simple validation check
```
if (!altinnAppGitRepository.AppUsesLayoutSets())
{
    // throw or whatever you're planning
}
```